application/models/bestitamazonpay4oxidbasketutil.php

```diff
@@ -70,7 +70,7 @@ public function restoreQuickCheckoutBasket()
     /**
      * Generates the basket hash.
      *
-     * @param string          $sAmazonOrderReferenceId
+     * @param string                                              $sAmazonOrderReferenceId
```

Comment:
What happend here?

Reply:
In the coding standadrd it is required that all arguments are in one row